src/AdminGuesser.test.js

```diff
@@ -12,14 +12,11 @@ describe('<AdminGuesser />', () => {
   test('renders errors', () => {
     const tree = renderer.render(
       <Provider store={store}>
-        {AdminGuesserComponent(
-          {},
-          {
-            error: 'Failed to fetch documentation',
-            data: {resources},
-            loading: false,
-          },
-        )}
+        {AdminGuesserComponent({
```

Comment:
I think it has been done this way because the `ShallowRenderer` from `react-test-renderer` only allows to render the component "one level deep" and using the `Provider` is necessary.

Reply:
You mean in tests? Then don't use shallow rendering, use react-testing-library instead.

Reply:
The `Provider` was not needed anymore so we can keep the `react-test-renderer`.